tests/test_apps.py

```diff
@@ -33,6 +34,19 @@ def test_get_jwt(self, time_mock):
 
         assert result == jwt.encode(expected_payload, private_key, algorithm="RS256")
 
+        # Custom expiration
+        expiration = 30 * 60
```

Comment:
My preference would be either to test this as a different test case, (maybe by adding `test_get_jwt_with_custom_expiry` or as a parametrized test.

Reply:
I did it as a separate test case, so that the default value of the `expiration` parameter is tested in the original test case.